pkg/scenarioorchestrator/common_functions.go

```diff
@@ -64,6 +82,13 @@ func CommonRunGraph(
 				volumes[k] = v
 			}
 
+			// inject resiliency config as env variable if provided (Use Case: Resiliency Config is provided via env variable)
+			if scenario.ResiliencyConfigPath != "" {
+				if content, err := os.ReadFile(scenario.ResiliencyConfigPath); err == nil {
+					env["RESILIENCY_CONFIG"] = string(content)
```

Comment:
are you sure that passing the file content as it is will work in any condition? don't you think that serializing the file content in base64 for example might ensure that the file structure (including indentation, spacing quotes etc.) is safely kept?